core/iwasm/libraries/libc-wasi/sandboxed-system-primitives/src/posix.c

```diff
@@ -319,41 +279,43 @@ wasi_addr_ip_to_bh_ip_addr_buffer(__wasi_addr_ip_t *addr,
     }
 }
 
+bh_clock_id_t
+convert_clock_to_bh_clock(__wasi_clockid_t clock_id)
```

Comment:
I'd suggest we turn this function into the following:
```suggestion
bool
convert_clock_to_bh_clock(__wasi_clockid_t in, bh_clock_id_t *out)
```
That way we clearly indicate when there's an error.